common/app/model/pressedContent.scala

```diff
@@ -162,7 +162,7 @@ object LatestSnap {
       card = PressedCard.make(content),
       discussion = PressedDiscussionSettings.make(content),
       display = PressedDisplaySettings.make(content),
-      format = None,
+      format = Some(ContentFormat.fromFapiContentFormat(content.format)),
```

Comment:
[`LatestSnaps`](https://github.com/guardian/facia-tool/blob/main/docs/Glossary.md#snaplinks) are `Snaplinks (i.e. cards that are not a direct reference to article) that display the latest article available for a tag or a section at the time of pressing.

This commit passes CAPI format to `LatestSnaps` on the `facia-press` level because we want to be able to consume it in the platform and give the correct styling to the contributors cards. If CAPI format is not present the trail [will get the default](https://github.com/guardian/facia-scala-client/blob/main/fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala#L191) one `ContentFormat(ArticleDesign, NewsPillar, StandardDisplay)`. 

`Format` from CAPI and Facia clients was first introduced in DCR trails [in this PR](https://github.com/guardian/frontend/pull/23593). However `format` was set to `None` for `LatestSnaps` although [it got passed on the FAPI client level in `fapi.LatestSnap`](https://github.com/guardian/facia-scala-client/commit/9c9357773ff42ec235465355754e2f6392bf7a19).